src/Services/privateApiService.js

```diff
@@ -6,14 +6,24 @@ const config = {
     }
 }
 
+const privateInstance = axios.create({
+    baseURL: 'http://ongapi.alkemy.org/public/api',
+});
+
+privateInstance.interceptors.request.use((config)=> {
+    config.headers.Authorization = localStorage.getItem('token');
```

Comment:
please reuse the method created by @ArielSurco on the user story [Crear método que devuelva el encabezado de seguridad para ser reutilizado en las peticiones a los metodos privados de la API](https://alkemy-labs.atlassian.net/jira/software/c/projects/OT91/boards/145?modal=detail&selectedIssue=OT91-69)

Reply:
the function is on this file: getAuthorizationHeader

Reply:
Done